### PR TITLE
Remove fancy tab widget

### PR DESCRIPTION
**Pull request by mpuchner**
Remove any reference to Manhattan::FancyTabWidget.

Note that there are two points where high dpi devices might need special treatment. This has to be tested. The relevant points are marked with ```// TODO: check with High-DPI devices!```

### CODE REVIEW

File: GUI/coregui/mainwindow/mainwindow.cpp [hunk 1]
```diff
@@ -56,13 +64,46 @@ MainWindow::MainWindow()
...
+    vlayout->setMargin(0);
+    vlayout->setSpacing(0);
+    vlayout->addLayout(m_viewsStack);
+    vlayout->addWidget(m_statusBar);
+
+    mainLayout->addLayout(m_viewSelectionButtonsLayout);
+    mainLayout->addLayout(vlayout);
+
+    setCentralWidget(centralWidget);
+
```
Comment (gpospelov) [inline GUI/coregui/mainwindow/mainwindow.cpp]: In the future, I would extract code to separate class to have mainwindow relatively light.

File: GUI/coregui/mainwindow/mainwindow.cpp [hunk 2]
```diff
@@ -298,7 +298,7 @@ void MainWindow::addView(ViewId id, const QIcon& icon, const QString& title, con
     QToolButton* btn = createViewSelectionButton();
     m_viewSelectionButtonsLayout->insertWidget(id, btn);
 
-    btn->setFixedSize(70, 70); // #TODO: High-DPI?
+    btn->setFixedSize(70, 70); // TODO: check with High-DPI devices!
```
Comment (gpospelov) [inline GUI/coregui/mainwindow/mainwindow.cpp:301]: I suggest in the future to switch to something in the line
```
QSize FindSizeOfLetterM()
{
    QFontMetrics fontMetric(QApplication::font());
    auto em = fontMetric.horizontalAdvance('M');
    auto fontAscent = fontMetric.ascent();
    return QSize(em, fontAscent);
}

QSize ModelView::Utils::SizeOfLetterM()
{
    static QSize result = FindSizeOfLetterM();
    return result;
}

int ModelView::Utils::WidthOfLetterM()
{
    return ModelView::Utils::SizeOfLetterM().width();
}
```
and use this as a typical size for DPI sensitive things, so your code would become
```
btn->setFixedSize(WidthOfLetterM()*42, WidthOfLetterM()*42);
```

**Review by gpospelov — COMMENTED** [review]
Very nice. Visual glitches are present on Plasma+4k, 32" but this we fix later. Please see also my comments in code.

fancytab

**Comment by mpuchner** [comment]
Thanks for your comments! For high dpi I would appreciate your help, I don't have a 4k device :-(

**Pull request merged**